.github/workflows/gh-ci.yaml

```diff
@@ -0,0 +1,281 @@
+name: mda_gh_ci
```

Comment:
This might be part of our current offline discussion, but why are these showing up on the diff?

Reply:
Because I'm still a git noob!

Should be fixed now.